test/Feature/ConstExprWithDivZero.c

```diff
@@ -0,0 +1,17 @@
+// RUN: %llvmgcc %s -emit-llvm -g -O1 -c -o %t.bc
```

Comment:
It's seems for newer LLVM's the test isn't working. Likely the division is getting optimized away. Do you really need to build with `-O1`? Perhaps it would be better to write a simple test case directly in LLVM IR to by-pass Clang.